ztp/siteconfig-generator/siteConfig/siteConfigBuilder_test.go

```diff
@@ -1002,18 +1002,18 @@ spec:
   clusterImageSetNameRef: "openshift-v4.8.0"
   clusters:
   - clusterName: "cluster1"
+    diskPartition:
+      - device: /dev/sda
+        partitions:
+          - mount_point: /var/imageregistry
```

Comment:
Does it make sense to make this mount point name more generic (since an additional partition can be used for any purpose)? You have already refactored "image-registry" to "var-device" in the MC name, so it's only here